src/cli/commands/__tests__/__fixtures__/ruleset-extends-invalid.yaml

```diff
@@ -0,0 +1,11 @@
+extends:
```

Comment:
Extends is a bit overloaded with expectations from OOP, could we stick with "require" like we discussed in the issues and everywhere?

Reply:
Hey, I'm merging this to the `--ruleset` branch.
Let's continue discussion over there. I'm not sure about require. If you look at the implementation I actually do extend (merge) rulesets. So `require` is a bit misleading (isn't it?).